testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -121,6 +121,8 @@ impl RunLoop {
             Some(should_keep_running.clone()),
         );
         let pox_constants = burnchain.get_pox_constants();
+        // Initialize the wallet.
+        burnchain.create_wallet_if_dne();
```

Comment:
This should only be invoked if the Stacks node is running as a miner.  If you try this on a public Bitcoin node, it will fail.